src/routes/delegates/handlers.rs

```diff
@@ -46,7 +48,6 @@ pub async fn post_delegate(
         request.body(serde_json::to_string(&safe_delegate_create).ok());
         request
     };
-
     context.http_client().post(request).await?;
     Ok(())
 }
```

Comment:
We should discard the changes to this file:
- the `Debug` import is not required
- Removing the new newline should also not be part of this PR